mx.truffleruby/suite.py

```diff
@@ -4,15 +4,7 @@
 
     "imports": {
         "suites": [
-            {
-                "name": "truffle",
-                "subdir": True,
-                "version": "ac78c6f6119bed0175c07ed732b3e12b689b5d1c",
-                "urls": [
-                    {"url": "https://github.com/oracle/graal.git", "kind": "git"},
-                    {"url": "https://curio.ssw.jku.at/nexus/content/repositories/snapshots", "kind": "binary"},
-                ]
-            },
+            # Import only tools which depends on truffle
```

Comment:
Minor: I would add ` to avoid duplication` to clarify there is technical problem to import both.

Reply:
the comment exists only until #1258, not improving further